database/migrations/2022_06_05_153600_create_activity_log_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateActivityLogTable extends Migration
```

Comment:
Is it possible to make these migrations all anonymous?

```suggestion
return new class extends Migration
```